From c853a932ea53d3bec9e4509d263245421b07af4d Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Fri, 25 Nov 2016 22:48:48 -0700
Subject: [PATCH 11/11] wait_ms fix

---
 include/usb.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/usb.h b/include/usb.h
index 97ea37b..d3acbd5 100644
--- a/include/usb.h
+++ b/include/usb.h
@@ -197,7 +197,7 @@ int usb_submit_int_msg(struct usb_device *dev, unsigned long pipe,
 			void *buffer, int transfer_len, int interval);
 void usb_disable_asynch(int disable);
 int usb_maxpacket(struct usb_device *dev, unsigned long pipe);
-inline void wait_ms(unsigned long ms);
+extern inline void wait_ms(unsigned long ms);
 int usb_get_configuration_no(struct usb_device *dev, unsigned char *buffer,
 				int cfgno);
 int usb_get_report(struct usb_device *dev, int ifnum, unsigned char type,
-- 
2.10.2

--- a/include/image.h	2017-11-21 10:52:40.000000000 -0500
+++ b/include/image.h	2020-04-18 23:00:03.360000000 -0400
@@ -591,9 +591,9 @@
 #define FIT_MAX_HASH_LEN	20	/* max(crc32_len(4), sha1_len(20)) */
 
 /* cmdline argument format parsing */
-inline int fit_parse_conf (const char *spec, ulong addr_curr,
+extern inline int fit_parse_conf (const char *spec, ulong addr_curr,
 		ulong *addr, const char **conf_name);
-inline int fit_parse_subimage (const char *spec, ulong addr_curr,
+extern inline int fit_parse_subimage (const char *spec, ulong addr_curr,
 		ulong *addr, const char **image_name);
 
 void fit_print_contents (const void *fit);
